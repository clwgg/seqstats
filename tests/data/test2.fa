>a
ACGT
>b
ACGTAC
>c
A
>d
AC
>e
ACGTA

