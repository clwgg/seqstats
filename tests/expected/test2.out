Total n:	5
Total seq:	18 bp
Avg. seq:	3.60 bp
Median seq:	4.00 bp
N 50:		5 bp
Min seq:	1 bp
Max seq:	6 bp

